components/widgets/loading/loading_bars.jsx

```diff
@@ -20,10 +18,7 @@ export default class LoadingBars extends React.PureComponent {
 
         return (
             <span className={'LoadingBars' + (text ? ' with-text' : '')}>
-                <img
-                    className='spinner'
-                    src={loadingGif}
-                />
+                <i className='fa fa-spin fa-spinner'/>
```

Comment:
This is changing the loading bars everywhere we were using loading bars, which is not just during the preview loading. That is the intention? If that is the intention, I would expect to change the name to other thing, and actually if we going to use a "spinner", I would prefer to use the "LoadingSpinner" component instead the loadingBars.

My proposal here would be us LoadingSpinner for image preview instead of loading bars, and leave LoadingBars as is now.

Reply:
Done @jespino Good catch.